app/compile-validators.hs

```diff
@@ -51,7 +51,7 @@ main = do
   writePlutusScript scriptnum "interest.plutus" (Interest.interestScript (Interest.ContractInfo getLenderNftCs)) (interestShortBs (Interest.ContractInfo getLenderNftCs))
   writePlutusScript scriptnum "collateral.plutus" (Collateral.collateralScript getCollateralScParams) (collateralShortBs getCollateralScParams)
   writePlutusScript scriptnum "request.plutus" (Request.request getRequestScParams) (requestShortBs getRequestScParams)
-  writePlutusScript scriptnum "liquidation.plutus" Liquidation.liquidation liquidationShortBs
+  writePlutusScript scriptnum "liquidation.plutus" (Liquidation.liquidation $ Liquidation.ContractInfo getBorrowerNftCs) (liquidationShortBs  $ Liquidation.ContractInfo getBorrowerNftCs)
```

Comment:
The currency symbol is the same for both borrower and lender NFT. I think similar change was there somewhere in a previous PR already. So just pointing out that we could unify it here as well
```suggestion
  writePlutusScript scriptnum "liquidation.plutus" (Liquidation.liquidation $ Liquidation.ContractInfo getAadaNftCs) (liquidationShortBs  $ Liquidation.ContractInfo getAadaNftCs)
```

Reply:
You can ignore the comment, I noticed that this was actually updated in [here](https://github.com/aadafinance/aada-finance/pull/29)